src/main/java/org/broadinstitute/hellbender/tools/spark/bwa/BwaSpark.java

```diff
@@ -25,19 +25,12 @@
 
     public static final String SINGLE_END_ALIGNMENT_FULL_NAME = "singleEndAlignment";
     public static final String SINGLE_END_ALIGNMENT_SHORT_NAME = "SE";
-    public static final String BWA_MEM_INDEX_IMAGE_FULL_NAME = "bwaMemIndexImage";
-    public static final String BWA_MEM_INDEX_IMAGE_SHORT_NAME = "image";
 
     @Argument(doc = "the output bam",
             shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
             fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME)
     private String output;
 
-    @Argument(doc = "the bwa mem index image file name that you've distributed to each executor",
-            fullName = BWA_MEM_INDEX_IMAGE_FULL_NAME,
-            shortName = BWA_MEM_INDEX_IMAGE_SHORT_NAME )
-    private String indexImageFile;
```

Comment:
Maybe the indexImageFile can be an optional argument; if not present, the default img file will be used, otherwise use the one provided. Thus, this requires to move the code for detecting the image to the tool itself. Is that what you mean, @tedsharpe?